YoCode/TestCountCheck.cs

```diff
@@ -40,7 +42,7 @@ public void ExecuteTheCheck()
             tempStats = StatLine.GetNumbersInALine();
             StoreCalculations(tempStats);
 
-            UnitTestEvidence.FeatureImplemented = stats.percentagePassed == 100 ? true :false;
+            UnitTestEvidence.FeatureImplemented = stats.percentagePassed == 100 && stats.totalTests > TestCountTreshold ? true :false;
```

Comment:
I don't think you need the conditional operator here (the `?:` ternary operator) because `stats.percentagePassed == 100 && stats.totalTests > TestCountTreshold` is already boolean.

Reply:
I was trying to be smart :(